compliant/src/main.rs

```diff
@@ -0,0 +1,155 @@
+use anyhow::{Context, Result};
```

Comment:
Small nit: we use `eyre` in `build2cmake` and `kernel-abi-check`. Maybe we should use the same here for consistency?

Reply:
good catch, i've updated to prefer eyre over anyhow in the latest changes. Thanks!